.remarkrc.js

```diff
@@ -1,10 +1,8 @@
-'use strict';
-
-module.exports = {
+export default {
 	plugins: [
-		require('remark-frontmatter'),
-		require('remark-preset-lint-recommended'),
-		require('remark-validate-links'),
-		require('remark-preset-prettier'),
+		'remark-frontmatter',
+		'remark-preset-lint-recommended',
+		'remark-validate-links',
+		'remark-preset-prettier',
```

Comment:
See https://github.com/JounQin/remark-preset-prettier/issues/61#issuecomment-898907897

Reply:
@JounQin Thanks. I'll look into it more.

@jeddy3 It seems this PR doesn't work yet.

Reply:
You need `import` related plugins like `require` as previous.

Reply:
@JounQin Thank you. I've fixed it via b375253.